rero_ils/query.py

```diff
@@ -68,22 +70,27 @@ def inner(values):
     return inner
 
 
-def and_i18n_term_filter(field):
-    """Create a i18n term filter.
+def and_i18n_term_filter(field, **kwargs):
+    """Create an i18n term filter.
 
     :param field: Field name.
+    :param kwargs: additional sub-filters to apply on (allowed keys are 'must'
+        or 'must_not')
     :return: Function that returns a boolean AND query between term values.
     """
     def inner(values):
-        language = request.args.get("lang", current_i18n.language)
+        language = request.args.get('lang', current_i18n.language)
         if not language or language not in get_i18n_supported_languages():
             language = current_app.config.get('BABEL_DEFAULT_LANGUAGE', 'en')
-        i18n_field = '{field}_{language}'.format(
-            field=field,
-            language=language
-        )
+        i18n_field = f'{field}_{language}'
         must = [Q('term', **{i18n_field: value}) for value in values]
```

Comment:
We can probably use a term set filters/query: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-terms-set-query.html